flocker/node/test/test_loop.py

```diff
@@ -724,6 +726,43 @@ def test_status_update_while_sleeping_no_discovery(self):
             remaining_discover_calls - len(self.deployer.local_states),
             0)
 
+    def test_longer_sleep_when_converged(self):
+        """
+        When a convergence loop results in a ``NoOp`` the sleep is based on
+        that configured on the ``Deployer``.
+        """
```

Comment:
Nothing leaped to mind at the time. I'll think about it more. Any suggestions would be welcome.